app/views/layouts/_topbar.html.haml

```diff
@@ -0,0 +1,26 @@
+%nav.top-bar{"data-topbar" => "", :role => "navigation"}
+  %ul.title-area
+    %li.name
+      %h1
+        %a{:href => "#"} Shostakodech
+    / Remove the class "menu-icon" to get rid of menu icon. Take out "Menu" to just have icon alone
+    %li.toggle-topbar.menu-icon
+      %a{:href => "#"}
+        %span Menu
+  %section.top-bar-section
+    / Right Nav Section
+    %ul.right
+      %li.active
+        %a{:href => "#"} Right Button Active
+        - if user_signed_in?
+          %li= link_to 'Sign Out', destroy_user_session_path, method: :delete
+        - else
+          %li.has-dropdown
+            %a{:href => "#"} Account
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace